compiler/rustc_hir_analysis/src/errors/wrong_number_of_generic_args.rs

```diff
@@ -888,7 +888,7 @@ impl<'a, 'tcx> WrongNumberOfGenericArgs<'a, 'tcx> {
         let comma = if args.len() > 0 { ", " } else { "" };
         let trait_path = self.tcx.def_path_str(trait_def_id);
         let method_name = self.tcx.item_name(self.def_id);
-        err.span_suggestion(
+        err.span_suggestion_verbose(
```

Comment:
Did you forget this one?

Reply:
Those correspond to these, https://github.com/rust-lang/rust/pull/127374/files/921de9d8eae16947c375febe0ab8709797b37119#diff-1d620ac138e59caa4ee1265c976245aea8bb80648f969e59af382d38bfd84dd9R14, which I could not find an example where they would be rendered on their own, but given how *long* the suggested code and span are, I'd prefer to leave these as verbose. (I personally find "help: frobnicate the expression: `very::long::code::<that, goes, const { on }>(and, on)`" to be quite hard to read.)